test_setup.wiki_tarball.yml

```diff
@@ -0,0 +1,56 @@
+---
+# This playbooks downloads tarball with a dump of all english wikipedia
+# articles {{ wiki_tarball_filename }} on a client machine, so that
+# one can try to use it for testing, eg:
+#
+#  * extract some articles into individual files on gluster volume via
+#  wiki-export-split.py script, distributing lot of small files on all bricks
+#  so that the files on the bricks will be distributed uniformly
+#
+#  * copy the tarball into the volume directly (this will be a bit faster than
+#  generating equivalent size of random data)
+
+- hosts: usm_client
+  remote_user: root
+  vars:
+    - wiki_tarball_filename: "enwiki-latest-pages-articles.xml.bz2"
+    - wiki_tarball_dest: "/tmp"
+
+  tasks:
+
+    - name: Check that mandatory variables are defined
+      assert:
+        that:
+          - wiki_tarball_url is defined
+          - wiki_tarball_checksum is defined
+        msg:
+          - "You need to define *url of the tarball* in wiki_tarball_url"
+          - "variable in your invenory file."
+          - "Don't use original source, but host the file on your own machine!"
+          - "Eg. http://example.com/enwiki-latest-pages-articles.xml.bz2"
+          - ""
+          - "You also need to define *checksum of the tarball* in"
+          - "wiki_tarball_checksum variable."
+          - "Eg. md5:a538459bfe1bc35e4d8161840eadbcd"
+
+    - name: Download the wiki tarball into /tmp directory
```

Comment:
Name hints that it is downloaded to /tmp but destination is variable. It would be good to rename it to something like: wiki_tarball_dest (default is /tmp).

Reply:
The point of this task name is to communicate where the tarball will be stored, moreover you are not expected to change the wiki_tarball_dest variable. I will add a variable name into the name though.